packages/apps/immutable-records/build.mjs

```diff
@@ -0,0 +1,36 @@
+import { spawn } from 'child_process';
+
+/*
+ * NextJS build outputs a depredaction warning related to
+ * usage of @kadena/react-ui/theme. It originated from webpack5
+ * with the mini-css-extract-plugin.
+ *
+ * We have investiated the issue and even when upgrading packages
+ * to the latest versions the warning remains.
+ *
+ * This file is a workaround to mute the warning until newer versions
+ * are available that fix the issue.
+ */
+
+const stderrFilter = [
+  '[DEP_WEBPACK_MODULE_UPDATE_HASH] DeprecationWarning',
+  'node --trace-deprecation',
+];
+
+function muteStderrWarning(stderr) {
+  return stderr
+    .toString()
+    .split('\n')
+    .filter((line) => !stderrFilter.find((warning) => line.includes(warning)))
+    .join('\n');
+}
+
+const build = spawn('node_modules/.bin/next', ['build']);
+
+build.stdout.on('data', (data) => {
+  process.stdout.write(data);
+});
+
+build.stderr.on('data', (data) => {
+  process.stderr.write(muteStderrWarning(data));
+});
```

Comment:
Warnings are annoying, but not worth a workaround like this. Do we need this to let the build succeed in CI? I have a hunch this is caused by https://github.com/kadena-community/kadena.js/blob/main/common/config/rush/command-line.json#L165-L185

Reply:
Ah, that does look like exactly the properly we would need to solve this! indeed much better then a custom startup script.

I am wondering if there is a way to enable this for only the "@kadena/immutable-records" app? From what I see this is configured once for the whole repo. That would not be my intention.

Would you rather have this setting changed for the `build` command in the entire repo, or keep the single-app script?

Reply:
Looking at it more thoroughly, there doesn't seem to be a build command in `commnad-line.json`, so I'm not sure how `allowWarningsInSuccessfulBuild` would be added.